### DISCUSSION TITLE
Introduce fastfetch, a fast, cross platform, neofetch like system info tool

### DISCUSSION THREAD

**Discussion by CarterLi** [lptstr/winfetch#170] [Show and tell]
https://github.com/LinusDierheimer/fastfetch

## Windows Native:

![image](https://user-images.githubusercontent.com/6134068/205117230-8ff1c4ea-ffbc-4bd1-adf9-3473367bebf9.png)

## WSL2

![image](https://user-images.githubusercontent.com/6134068/205118434-2cf9b46b-9dea-43d8-aae4-7f6f789f087f.png)

## Some benchmark results

![image](https://user-images.githubusercontent.com/6134068/205119161-e2e053b9-6b30-41d5-9676-ad80e9346e78.png)

https://github.com/LinusDierheimer/fastfetch/wiki/Fastfetch-VS-Winfetch-(Windows)

PS: The Windows part, coded by [me  (a former winfetch user)](https://github.com/lptstr/winfetch/issues/146), is highly inspired by winfetch. Winfetch is really good and high quality project. Thanks a lot.

**Comment by rashil2000** [comment]
Very nice, starred! Make sure to create a PR in the Scoop [Main bucket](https://github.com/ScoopInstaller/Main) once the project reaches 500 stars 😁

There's also [macchina](https://github.com/Macchina-CLI/macchina) which focuses on performance, but its Windows support is, well, not a priority 😅
